internal/controller/driver_controller.go

```diff
@@ -741,7 +823,26 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 								),
 							})
 						}
-
+						// CSI Logrotate Container
+						if logRotator != nil {
+							resources := ptr.Deref(pluginSpec.Resources.LogRotator, corev1.ResourceRequirements{})
+							containers = append(containers, corev1.Container{
+								Name:            "log-collector",
```

Comment:
Why collector and not rotator? I don't see an need to change established terminology here
```suggestion
								Name:            "log-rotator",
```